### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by jjaykim**
# mobile version `league standing table` 
### display only `Position` `Club` `Played` `Points` columns
Screen Shot 2021-11-10 at 9 19 59 AM

# mobile version `league schedule table`
Screen Shot 2021-11-10 at 11 49 16 AM
`

### CODE REVIEW

File: client/src/pages/League.tsx [hunk 1]
```diff
@@ -41,7 +41,6 @@ const UnstyledLeague: FunctionComponent<LeagueProps> = ({ className }) => {
 
   if (!league || !viewer.leagueTeamGroupAge || !viewer.matchesByAge)
     return <div>Loading...</div>;
-  // margin-left: 7rem;
```
Comment (koreancanadiansoccer) [inline client/src/pages/League.tsx:44]: remove

File: client/src/components/league_schedule/LeagueSchedule.tsx [hunk 2]
```diff
@@ -16,52 +16,128 @@ import dayjs from 'dayjs';
...
+          </Box>
+        ),
+
+        VS: (
+          <Box
+            className="mobile-vs"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            width={'80%'}
```
Comment (koreancanadiansoccer) [inline client/src/components/league_schedule/LeagueSchedule.tsx:63]: no need to wrap this?
width='80%'

File: client/src/pages/League.tsx (hunk 1 above)
Comment (jjaykim) [reply client/src/pages/League.tsx:44]: I think I already removed it lol

File: client/src/components/league_schedule/LeagueSchedule.tsx [hunk 3]
```diff
@@ -16,52 +16,128 @@ import dayjs from 'dayjs';
...
+            />
+            <Box fontSize={14} ml={1}>
+              {match.awayTeam.team.name.toUpperCase()}
+            </Box>
+          </Box>
+        ),
+      },
+    ];
+  }
+  else {
```
Comment (koreancanadiansoccer) [inline client/src/components/league_schedule/LeagueSchedule.tsx:104]: do you need else here?

**Review by koreancanadiansoccer — APPROVED** [review]

**Review by koreancanadiansoccer — APPROVED** [review]
approving for now

File: client/src/components/league_schedule/LeagueSchedule.tsx (hunk 3 above)
Comment (jjaykim) [reply client/src/components/league_schedule/LeagueSchedule.tsx:104]: I think no need it lol

File: client/src/components/league_schedule/LeagueSchedule.tsx (hunk 2 above)
Comment (jjaykim) [reply client/src/components/league_schedule/LeagueSchedule.tsx:63]: wrap이요...?

**Comment by koreancanadiansoccer** [comment]
fixed - merging it.

**Pull request merged**